File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.py
def solution(sides):
    # s=sorted(sides)
    # if longest side is s[-1] and if longest side is the rest
    # (s[-1]-(s[-1]-s[0]))+(s[-1]+s[0]-s[-1]-1)
    r=2*min(sides)-1
    return r

File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// sides_len은 배열 sides의 길이입니다.
int solution(int sides[], size_t sides_len) {
    int r=0,min=sides[0];
    min=min>sides[1]?sides[1]:min;
    r=2*min-1;
    return r;
}